docs/source/globus_action_providers.rst

```diff
@@ -408,3 +408,371 @@ should be used in the Datacite test service or the production service.
 .. _Globus Transfer Directory API: https://docs.globus.org/api/transfer/file_operations/#list_directory_contents
 .. _Globus Transfer Make Directory API: https://docs.globus.org/api/transfer/file_operations/#make_directory
 .. _Globus Transfer Get Collection API: https://docs.globus.org/api/transfer/endpoint/#get_endpoint_by_id
+
+
+HTTP requests
+-------------
+
+URL: `<https://actions.globus.org/http>`_
+
+Scope: ``https://auth.globus.org/scopes/5fac2e64-c734-4e6b-90ea-ff12ddbf9653/http``
+
+Synchronous / Asynchronous: Synchronous
+
+The HTTP action provider supports making HTTP requests to arbitrary URL's of your choosing.
+This allows you to gather information from, or submit information to, external services.
+
+At minimum, you must specify the HTTP method and the target URL.
+Here is an example of the simplest functional input document:
+
+..  code-block:: json
+
+    {
+        "method": "GET",
+        "url": "https://httpbin.org/get"
+    }
+
+A single HTTP request can be customized in four key ways:
+
+*   HTTP method
+*   HTTP timeout
+*   Content submission
+*   URL customization
+
+..  note::
+
+    The JSON schema that authoritatively describes the accepted keys and values can be retrieved using this command:
+
+    ..  code-block:: text
+
+        globus-automate action introspect --action-url https://actions.globus.org/http
```

Comment:
Since this isn't specific to this AP I think it makes sense to avoid duplication in favor of what's at the top of the file. It could be that the text at the top of the file could use reformatting/updating to make it more clear that it applies to every listed AP.

Reply:
100% agree. Should that be in the scope of this work? If not I'll create a story.

Reply:
If you have any immediate thoughts on how to make the text on introspection [at the top of the doc here](https://github.com/globus/globus-automate-client/blob/master/docs/source/globus_action_providers.rst) more visible then I'd say we can make the changes here on this PR. If not then let's cut the duplicated instructions here and keep an eye out for a potential doc update in the future. From my experience doc work can be a deep rabbit hole and I don't think a broad update is necessary right now.

Reply:
Oh! I just realized that you're pointing out that the instructions are already documented at the top of the doc. :disappointed_relieved: Okay, I'll get this updated.

Reply:
Done. Thanks for pointing this out!